network/src/p2p/builder.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+use super::*;
+use libp2p::{
+    core::{muxing::StreamMuxerBox, transport, upgrade},
+    futures::{AsyncRead, AsyncWrite},
+    identity,
+    noise::{self, AuthenticKeypair, X25519Spec},
+    pnet::PnetConfig,
+    PeerId, Transport,
+};
+use libp2p_mplex::MplexConfig;
+use std::{borrow::Borrow, time::Duration};
+
+/// Builds libp2p transport for mina with various configurations
+#[derive(Clone)]
+pub struct MinaTransportBuilder {
```

Comment:
naming: can we remove the mina prefix and just have `TransportBuilder`? reason: the project name is likely to change. likewise for other entities wherever possible.

Reply:
Fixed